Dockerfile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+FROM quay.io/kroxylicious/eclipse-temurin:20-jdk AS builder
```

Comment:
should we use JDK-17 here if we are going to use `JAVA_VERSION=17`?

Reply:
The build requires 20 currently to do `mvn clean verify -Pdist -Dquick` since it compiles `integrationtests`.

Reply:
Maybe better to run `mvn clean verify -Pdist -Dquick -pl :kroxylicious -am` instead? building even less stuff that way

Reply:
I've switched to build with ubi9 and JDK 17, just building kroxylicious and it's dependencies.